components/script/dom/domimplementation.rs

```diff
@@ -80,7 +80,7 @@ impl DOMImplementationMethods for DOMImplementation {
         let doc = XMLDocument::new(win,
                                    HasBrowsingContext::No,
                                    None,
-                                   self.document.origin().alias(),
```

Comment:
The Rust naming contention seems to be that `clone` copies the reference to `Rc`.